events/static/events/create_event.css

```
@@ -29,7 +29,8 @@ a {
 .container {
   position: absolute;
   width: 600px;
-  /* height: 1100px; */
+  height: 900px;
+  overflow: auto;
```

```suggestion
  height: 85%;
  overflow: auto;
```

Maybe you can use a percentage instead of px, 
I have tried it on my computer with a variation of the site size and I think it looks better that way.
** in line 22 the value needs to be exactly as line 32

---

Thanks, done 😄